File: exercicios/Python/1071/1071.py
n1 = int(input());
n2 = int(input());

maior = n1 if n1 > n2 else n2;
menor = n2 if n2 < n1 else n1;

menor += 1;
soma = 0;
while (menor < maior):
    if(menor % 2 )!=0:
        soma += menor
    menor += 1;
print(soma)

File: exercicios/R/1071/1071.r
input = file('stdin','r')

n1 = as.integer(readLines(input,n=1))
n2 = as.integer(readLines(input,n=1))

maior = as.integer(0)
menor = as.integer(0)
soma = as.integer(0)

if(n1 > n2){
    maior = n1;
    menor = n2;
}else
{
    maior = n2;
    menor = n1;
}

    menor = menor + 1;

while(menor < maior){
    if(menor %% 2 != 0){
        soma =soma + menor;
    }
        menor =menor + 1;
}
write(soma,'')